templates/slim/block_image.html.slim

```diff
@@ -1,10 +1,11 @@
-.imageblock(id=@id class=[@style,role]
-    style=[("text-align: #{attr :align}" if attr? :align),("float: #{attr :float}" if attr? :float)].compact.join('; '))
-  .content
-    - if attr? :link
-      a.image href=(attr :link)
+- unless attr(1) == 'background' || attr(1) == "canvas"
```

Comment:
Python spoiled me on this. I'm a former Perl dev and I understand totally the right quoting for the right context.